mdsuite/project/project.py

```diff
@@ -130,16 +132,18 @@ def add_experiment(
         units: Union[str, Units] = None,
         cluster_mode: bool = None,
         active: bool = True,
-        data: Union[str, list, dict] = None,
+        fname_or_file_processor: Union[
+            str, mdsuite.file_io.file_read.FileProcessor, list
+        ] = None,
```

Comment:
Does this support pathlib as well, see #302

Reply:
probably at the moment it even does, but it is not stated clearly. I will include the explicit option